index.js

```diff
@@ -53,6 +53,19 @@ function fastifySensible (fastify, opts, next) {
     }
   }
 
+  if (opts?.sharedSchemaId) {
+    fastify.addSchema({
+      $id: opts?.sharedSchemaId,
```

Comment:
nit: `sharedSchemaId` is already defined inside the block

```suggestion
      $id: opts.sharedSchemaId,
```

Reply:
Fixed :)